tests/board.test.js

```diff
@@ -0,0 +1,51 @@
+const expect = require('chai').expect;
+const Board = require('../lib/board.js');
+
+it('is empty', () => {
```

Comment:
Maybe the test description could provide also the context?
ex: it initialises a new empty board?